src/main/java/hu/dpc/phee/operator/importer/RecordParser.java

```diff
@@ -156,6 +150,12 @@ public void addVariableToEntity(DocumentContext newVariable, String bpmnProcessI
                 variableParser.getBatchParsers().get(name).accept(Pair.of(batch, value));
                 batchRepository.save(batch);
 
+                if (name.equals("duplicateTransactionFile")) {
```

Comment:
Can we make this a bit more generic? As failedTransactionFile? And a field containing the reason?

This will help the same code to be used for multiple failure types as required in the future. We discussed this on the first day.

Reply:
updated to `failedTransactionFile`.